src/app/modules/common-regobs-api/models/obs-locations-response-dto-v2.ts

```diff
@@ -3,6 +3,7 @@ import { LatLngObject } from './lat-lng-object';
 export interface ObsLocationsResponseDtoV2 {
   Distance?: number;
   GeoHazardId?: number;
+  Description?: string;
```

Comment:
Her har du endret i en automatgenerert fil. Du må enten legge til dette feltet i API:et (jeg finner ingen PR for det), eller løse dette på en annen måte. Jeg skjønner ikke helt hvorfor dette skal løses i akkurat denne filen?

Reply:
vi må hente Description felt fra datrabasen for den indikerer om det var bruker som satt opp navn på lokasjonen (LocationName) eller om det skjedde automatisk (Description). Det er eneste måte å finne ut om brukeren kan redigere navn på sine egne lokasjoner.

Reply:
Sjekk PRen i api https://dev.azure.com/NVE-devops/Varsom%20Regobs/_git/regobs/pullrequest/4010?path=/Pipelines/DeployTemplates/deploy-web-api5-template.yml